org.sonarlint.eclipse.core/src/org/sonarlint/eclipse/core/internal/jobs/AnalyzeProjectJob.java

```diff
@@ -276,31 +277,27 @@ private void updateMarkers(Map<IResource, List<Issue>> issuesPerResource, Analys
     }
 
     Set<IFile> failedFiles = result.failedAnalysisFiles().stream().map(f -> f.<IFile>getClientObject()).collect(Collectors.toSet());
-    for (Entry<IResource, List<Issue>> resourceEntry : issuesPerResource.entrySet()) {
-      IResource resource = resourceEntry.getKey();
-      if (failedFiles.contains(resource)) {
-        continue;
-      }
-      if (resource instanceof IFile) {
-        trackIssues(resource, resourceEntry.getValue(), triggerType);
-      } else {
-        // TODO handle non-file-level issues
-      }
+    Map<IResource, List<Issue>> successfulFiles = issuesPerResource.entrySet().stream()
+      .filter(e -> !failedFiles.contains(e.getKey()))
+      // TODO handle non-file-level issues
+      .filter(e -> e.getKey() instanceof IFile)
+      .collect(Collectors.toMap(Entry::getKey, Entry::getValue));
+
+    trackIssues(successfulFiles, triggerType);
+    if (shouldUpdateServerIssues(triggerType)) {
+      trackServerIssues(issuesPerResource.keySet());
     }
   }
 
-  private void trackIssues(IResource resource, List<Issue> rawIssues, TriggerType triggerType) throws CoreException {
-    IProject project = resource.getProject();
-    String localModuleKey = project.getName();
-    SonarLintCorePlugin.getDefault().getModulePathManager().setModulePath(localModuleKey, project.getLocation().toString());
-    String relativePath = resource.getProjectRelativePath().toString();
+  private void trackIssues(Map<IResource, List<Issue>> rawIssuesPerResource, TriggerType triggerType) throws CoreException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "triggerType". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)